backend/sozisel/lib/sozisel/model/quizzes/quiz_result.ex

```diff
@@ -5,19 +5,21 @@ defmodule Sozisel.Model.Quizzes.QuizResult do
   alias Sozisel.Model.Quizzes.ParticipantAnswer
 
   @type t :: %__MODULE__{
-          participant_answers: [ParticipantAnswer.t()]
+          participant_answers: [ParticipantAnswer.t()],
+          quiz_answer_time: Float.t()
         }
 
   @primary_key false
 
   embedded_schema do
     embeds_many :participant_answers, ParticipantAnswer, on_replace: :delete
+    field :quiz_answer_time, :float
```

Comment:
We already know that it is a quiz right? So `answer_time` should be enough (unless we are matching this model somewhere based on this specific field name)